master.cfg

```diff
@@ -226,16 +226,18 @@ if tcmalloc_slaves:
         factory=hc_factory))
 
 doc_slaves = slaves.GetSlaves(config_helper.GenerateDoc)
-c['builders'].append(
-    BuilderConfig(name="doxygen-doc-generator",
-      slavenames=[s.name() for s in doc_slaves],
-      factory=doc_factory))
+if doc_slaves:
+  c['builders'].append(
+      BuilderConfig(name="doxygen-doc-generator",
+        slavenames=[s.name() for s in doc_slaves],
```

Comment:
Should probably fix the indent here next time (slavenames should be aligned under name)